app/assets/stylesheets/general.scss

```diff
@@ -2,6 +2,92 @@
 $error-red: #ff0101;
 $error-red-transparent: rgba(255, 1, 1, 0.5);
 
+/* General */
+body {
+  display: flex;
+  flex-direction: column;
+
+  .content-container {
+    width: 900px;
+    padding: 10px;
+    border-color: black;
+    border-width: 1px;
+    align-self: center;
+  }
+}
+
+.main-title {
+  font-size: 24px;
+}
+
+.list {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  flex:1;
+  padding: 0 0 10px 0;
+
+  .book-title {
+    font-weight: bold;
+  }
+
+  .field {
+    display: flex;
+    flex: 1;
+  }
+}
+
+.list-header {
+  border-bottom: 2px;
```

Comment:
You can combine all these three attributes into the following:
`border: 2px solid black` or `border: solid 2px black`, as you wish.